Dragonflight/WarriorArms.lua

```diff
@@ -337,6 +341,11 @@ spec:RegisterAuras( {
         duration = 8,
         max_stack = 1
     },
+    rallying_cry = {
+        id = 97463,
+        duration = function () return 10 + ( talent.inspiring_presence.enabled and 3 or 0 ) end,
```

Comment:
Agree with adding Rallying Cry's buff, you may want to add `shared = "player",` so that if another person's Rallying Cry is on you, it will be detected.  Otherwise, it will only detect your own Rallying Cry.

(The same applies to debuffs on targets.  If you wanted to know if *anyone's* Thunderclap were on your target, you'd put `shared = "target",` in Thunderclap's aura information.)

Reply:
Set all 3 specs rally to shared = "player". Nothing I can think off the top of my head where shared = "target" would make sense yet. Fix in https://github.com/Hekili/hekili/pull/1670/commits/bdb3a98a9cbf2ce5c6618bdb38bff09174714176